docs/docs/writing-your-own-enclave-host.md

```diff
@@ -30,26 +35,20 @@ public class MyEnclaveHost {
     }
 }
 ```
-
-Update the host build.gradle to reference it:
-
+3. Update the host build.gradle to reference the main class:
```

Comment:
Not sure how we did elsewhere and it is surely a minor thing, but should we markdown `build.gradle`?

Reply:
Done.